src/Files.App/Filesystem/Security/AccessMaskItem.cs

```diff
@@ -0,0 +1,51 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using System.ComponentModel;
+
+namespace Files.App.Filesystem.Security
+{
+	/// <summary>
+	/// Represents an access mask details, such as its name and changeability.
+	/// </summary>
+	public class AccessMaskItem : ObservableObject
+	{
+		public AccessMaskItem(AccessControlEntry ace, bool isEditable = true)
+		{
+			IsEditable = isEditable;
+
+			_ace = ace;
+			_ace.PropertyChanged += AccessControlEntry_PropertyChanged;
+		}
```

Comment:
The constructor should not be that high up in the file. It should be move after properties and fields.